src/test/scala/org/broadinstitute/dsde/workbench/sam/service/TosServiceSpec.scala

```diff
@@ -99,6 +99,24 @@ class TosServiceSpec extends AnyFlatSpec with TestSupport with BeforeAndAfterAll
     assert(getTosStatusResult.get, s"getTosStatus(${defaultUser.id}) should get the tos for the user")
   }
 
+  it should "allow a user to reject the ToS" in {
```

Comment:
Nit: The first half of this test is identical to the previous one. I would be tempted to either just delete have a single test point that both accepts and rejects (ie., this one), or share the duplicated between the 2 testpoints.